configs/CMakeLists.txt

```diff
@@ -57,6 +57,22 @@ if( VIAME_DOWNLOAD_MODELS )
     endif()
   endif()
 
+  # LEARN Related Models
+  if( VIAME_ENABLE_PYTORCH-CUTLER )
+    if( VIAME_DOWNLOAD_MODELS-POOLCAR )
+      DownloadAndExtract(
+        https://viame.kitware.com/api/v1/file/646b5abb4c19222431def881/download
+        ${VIAME_DOWNLOAD_DIR}/VIAME_PoolCar_10_shot_models-V1.0.zip
+        ${CMAKE_CURRENT_SOURCE_DIR}/pipelines )
+    endif()
+    if( VIAME_ENABLE_PYTORCH-CUTLER )
+      DownloadAndExtract(
+        https://viame.kitware.com/api/v1/item/646b5af78790390715ac66a7/download
+        ${VIAME_DOWNLOAD_DIR}/VIAME_PoolCar_10_shot_models-V1.0.zip
+        ${CMAKE_CURRENT_SOURCE_DIR}/pipelines )
+    endif()
+  endif()
```

Comment:
There's now a number of enable flags added by this branch:

VIAME_ENABLE_LEARN
VIAME_ENABLE_PYTORCH_CUTLER
VIAME_DOWNLOAD_MODELS-POOLCAR
VIAME_DOWNLOAD_MODELS-CONVNEXT

Just for complexity reductions sake, are all necessary? Or can 1-2 be reduced. For example, does having LEARN on but not POOLCAR or COVNEXT make no sense? Alternatively LEARN and CUTLER and they can be consolidated into the same build flag?

Also in the above example, the third if (VIAME_ENABLE_PYTORCH-CUTLER I'm assuming should be poolcar, the second if poolcar changed to convnext, and also the filename for the package download correct

Reply:
I ended up combining LEARN and CUTLER into one since they are the same. I think it makes sense keeping the downloads separate as there is a part of the cutler functionality that can be used without the models. I the links and variables should be correct now.